DESCRIPTION

```diff
@@ -12,7 +12,9 @@ Depends:
     tidymodels
 Suggests: 
     baguette,
+    bonsai,
```

Comment:
You should add these packages to the GHA workflow for dev versions: https://github.com/tidymodels/extratests/blob/main/.github/workflows/GH-R-CMD-check.yaml#L62:L81

Reply:
You got it! [327d9f4](https://github.com/tidymodels/extratests/pull/53/commits/327d9f4b051a80cc16912b235f1ff5783990b6ba)